doc-content/enterprise-only/project-data/preview-editor-latest-version-test-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='preview-editor-latest-version-test-proc']
+= Switching between versions of a test scenario
+
+{CENTRAL} provides you the ability to switch between the various versions of a test scenario. Every time you save the scenario, a new version of the scenario is listed under *Latest Versions*. To use this feature you should save the test scenario file at least once.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Latest Version*. All the versions of the file will be listed under *Latest Version* if they exist.
+. Click the version you want to work on. The selected version of the test scenario will open in the *Test Scenario (Preview)* editor.
+. From the editor toolbar, click *Restore*.
+. In the *Confirm Restore*,
+.. To add a comment, click *add a comment*.
+.. Click *Restore* to confirm.
+
+You will get a message stating that the selected version has been reloaded successfully in the editor.
```

Comment:
A message stating that the selected version has been reloaded successfully in the editor is displayed.

Reply:
Changed as suggested.